tasmota/tasmota_xsns_sensor/xsns_45_vl53l0x.ino

```diff
@@ -79,13 +79,17 @@ struct {
   uint16_t distance;
   uint16_t distance_prev;
   uint16_t buffer[5];
-  uint8_t ready = 0;
+  bool ready = false;
```

Comment:
Using integer implies integer values can occur. But this is clearly a true/false situation. There are some other bool variables in the code. Why do you see a difference here?

Reply:
We are maximizing the use of "sized int" as much as possible because we control the implementation size.
With `bool` I'm unsure if this is being implement as a single byte or as an `int` which takes up 4 bytes and may be using more because of alignment.
Can you prove it's using only 1 byte ?

Reply:
Totally agree. Using uint8 is a common way to ensure that only 8 bits are used. To be sure, I checked if bool behaves the same way. And I can confirm it is also only 8 bits.

![image](https://github.com/user-attachments/assets/b0042aea-6ed8-4406-b1c7-6688afce1b56)

Reply:
`AddLog(LOG_LEVEL_DEBUG, PSTR("Size of uint8_t: %d"), sizeof(uint8_t));
 AddLog(LOG_LEVEL_DEBUG, PSTR("Size of bool: %d"), sizeof(bool));
 AddLog(LOG_LEVEL_DEBUG, PSTR("Size of variable 'ready': %d"), sizeof(Vl53l0x_data[i].ready));`